languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -3676,6 +3676,18 @@ though, and has since been slightly modified)-->
     </rulegroup>
     <rulegroup id="SUR_ACCENT" name="Sur au lieu de Sûr">
       <url>http://bdl.oqlf.gouv.qc.ca/bdl/gabarit_bdl.asp?id=2232</url>
+      <rule default="temp_off">
+          <pattern>
+             <token postag="SENT_START"/>
+             <token>Bien</token>
+             <marker>
+                 <token>sur[e?s?]</token>
+             </marker>
+          </pattern>
+          <message>Voulez-vous écrire <suggestion>sûr</suggestion> ?</message>
```

Comment:
In general, messages have to give the user an idea why that is a mistake. They should never just be "Did you mean XYZ?". In this case, you could say `Le mot 'sur' est une préposition. Vouliez-vous dire <suggestion>bien sûr</suggestion>, au sens de 'certainement' ?` or something like that.